### PR TITLE
Update readme python commands

### PR DESCRIPTION
**Pull request by peterl-stripe**
@gkk-stripe could you review this?  It fixes the python commands in the readme.  There's more I'd like to do on the readme (including the changes you suggested), but I'm not sure when I will given my TAMM work, so it would be good to merge at least this.  Feel free to make your own PR on the Readme if you have ideas on improving it.

CC @teapot-co/tempest

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,9 +142,13 @@ Tempest was built by a team of Stanford PhDs---[Peter Lofgren](@plofgren) (lead
    ```
    import tempest_db
    c = tempest_db.TempestClient()
-   alice_id = c.nodes("user", "username = 'alice'")[0]
-   alice_book_ids = c.out_neighbors("has_read", alice_id)
-   alice_book_titles = c.multi_node_attribute("book", alice_book_ids, "title")
+   alice = tempest_db.Node("user", "alice")
+   client.out_neighbors("follows", alice)
```
Comment (gkk-stripe) [inline README.md:146]: You should be using `c` instead of `client` here.

**Comment by gkk-stripe** [comment]
I'll go ahead and merge this and create another PR to fix typos in the README.

File: README.md (hunk 1 above)
Comment (peterl-stripe) [reply README.md:146]: Good catch!  Or it might be more clear to rename `c` to `client`.

**Comment by peterl-stripe** [comment]
Thanks, getting the Readme proofread/tested by someone like you who is less familiar with this code will be great!

**Pull request merged**